keyboards/handwired/reclined/info.json

```diff
@@ -0,0 +1,87 @@
+        { "label": "k07", "matrix": [0, 7], "w": 1, "x": 7, "y": 0 },
+        { "label": "k08", "matrix": [0, 8], "w": 1, "x": 8, "y": 0 },
+        { "label": "k09", "matrix": [0, 9], "w": 1, "x": 9, "y": 0 },
+        { "label": "k0a", "matrix": [0, 10], "w": 1, "x": 10, "y": 0 },
+        { "label": "k0b", "matrix": [0, 11], "w": 1, "x": 11, "y": 0 },
+        { "label": "k10", "matrix": [1, 0], "w": 1, "x": 12, "y": 0 },
+        { "label": "k11", "matrix": [1, 1], "w": 1, "x": 13, "y": 0 },
+        { "label": "k12", "matrix": [1, 2], "w": 1, "x": 14, "y": 0 },
+        { "label": "k13", "matrix": [1, 3], "w": 1, "x": 15, "y": 0 },
+        { "label": "k14", "matrix": [1, 4], "w": 1, "x": 16, "y": 0 },
+        { "label": "k15", "matrix": [1, 5], "w": 1, "x": 17, "y": 0 },
+        { "label": "k16", "matrix": [1, 6], "w": 1, "x": 18, "y": 0 },
+        { "label": "k17", "matrix": [1, 7], "w": 1, "x": 19, "y": 0 },
+        { "label": "k18", "matrix": [1, 8], "w": 1, "x": 20, "y": 0 },
+        { "label": "k19", "matrix": [1, 9], "w": 1, "x": 21, "y": 0 },
+        { "label": "k1a", "matrix": [1, 10], "w": 1, "x": 22, "y": 0 },
+        { "label": "k1b", "matrix": [1, 11], "w": 1, "x": 23, "y": 0 },
+        { "label": "k20", "matrix": [2, 0], "w": 1, "x": 24, "y": 0 },
+        { "label": "k21", "matrix": [2, 1], "w": 1, "x": 25, "y": 0 },
+        { "label": "k22", "matrix": [2, 2], "w": 1, "x": 26, "y": 0 },
+        { "label": "k23", "matrix": [2, 3], "w": 1, "x": 27, "y": 0 },
+        { "label": "k24", "matrix": [2, 4], "w": 1, "x": 28, "y": 0 },
+        { "label": "k25", "matrix": [2, 5], "w": 1, "x": 29, "y": 0 },
+        { "label": "k26", "matrix": [2, 6], "w": 1, "x": 30, "y": 0 },
+        { "label": "k27", "matrix": [2, 7], "w": 1, "x": 31, "y": 0 },
+        { "label": "k28", "matrix": [2, 8], "w": 1, "x": 32, "y": 0 },
+        { "label": "k29", "matrix": [2, 9], "w": 1, "x": 33, "y": 0 },
+        { "label": "k2a", "matrix": [2, 10], "w": 1, "x": 34, "y": 0 },
+        { "label": "k2b", "matrix": [2, 11], "w": 1, "x": 35, "y": 0 },
+        { "label": "k30", "matrix": [3, 0], "w": 1, "x": 36, "y": 0 },
+        { "label": "k31", "matrix": [3, 1], "w": 1, "x": 37, "y": 0 },
+        { "label": "k32", "matrix": [3, 2], "w": 1, "x": 38, "y": 0 },
+        { "label": "k33", "matrix": [3, 3], "w": 1, "x": 39, "y": 0 },
+        { "label": "k34", "matrix": [3, 4], "w": 1, "x": 40, "y": 0 },
+        { "label": "k35", "matrix": [3, 5], "w": 1, "x": 41, "y": 0 },
+        { "label": "k36", "matrix": [3, 6], "w": 1, "x": 42, "y": 0 },
+        { "label": "k37", "matrix": [3, 7], "w": 1, "x": 43, "y": 0 },
+        { "label": "k38", "matrix": [3, 8], "w": 1, "x": 44, "y": 0 },
+        { "label": "k39", "matrix": [3, 9], "w": 1, "x": 45, "y": 0 },
+        { "label": "k3a", "matrix": [3, 10], "w": 1, "x": 46, "y": 0 },
+        { "label": "k3b", "matrix": [3, 11], "w": 1, "x": 47, "y": 0 }
```

Comment:
Layout data isn't correct here; probably an issue with the input that was provided.

![image](https://user-images.githubusercontent.com/18669334/148516069-e21d981f-2f9c-49b7-88fd-03fd35c80827.png)

```suggestion
        { "label": "k10", "matrix": [1, 0], "w": 1, "x": 0, "y": 1 },
        { "label": "k11", "matrix": [1, 1], "w": 1, "x": 1, "y": 1 },
        { "label": "k12", "matrix": [1, 2], "w": 1, "x": 2, "y": 1 },
        { "label": "k13", "matrix": [1, 3], "w": 1, "x": 3, "y": 1 },
        { "label": "k14", "matrix": [1, 4], "w": 1, "x": 4, "y": 1 },
        { "label": "k15", "matrix": [1, 5], "w": 1, "x": 5, "y": 1 },
        { "label": "k16", "matrix": [1, 6], "w": 1, "x": 6, "y": 1 },
        { "label": "k17", "matrix": [1, 7], "w": 1, "x": 7, "y": 1 },
        { "label": "k18", "matrix": [1, 8], "w": 1, "x": 8, "y": 1 },
        { "label": "k19", "matrix": [1, 9], "w": 1, "x": 9, "y": 1 },
        { "label": "k1a", "matrix": [1, 10], "w": 1, "x": 10, "y": 1 },
        { "label": "k1b", "matrix": [1, 11], "w": 1, "x": 11, "y": 1 },
        { "label": "k20", "matrix": [2, 0], "w": 1, "x": 0, "y": 2 },
        { "label": "k21", "matrix": [2, 1], "w": 1, "x": 1, "y": 2 },
        { "label": "k22", "matrix": [2, 2], "w": 1, "x": 2, "y": 2 },
        { "label": "k23", "matrix": [2, 3], "w": 1, "x": 3, "y": 2 },
        { "label": "k24", "matrix": [2, 4], "w": 1, "x": 4, "y": 2 },
        { "label": "k25", "matrix": [2, 5], "w": 1, "x": 5, "y": 2 },
        { "label": "k26", "matrix": [2, 6], "w": 1, "x": 6, "y": 2 },
        { "label": "k27", "matrix": [2, 7], "w": 1, "x": 7, "y": 2 },
        { "label": "k28", "matrix": [2, 8], "w": 1, "x": 8, "y": 2 },
        { "label": "k29", "matrix": [2, 9], "w": 1, "x": 9, "y": 2 },
        { "label": "k2a", "matrix": [2, 10], "w": 1, "x": 10, "y": 2 },
        { "label": "k2b", "matrix": [2, 11], "w": 1, "x": 11, "y": 2 },
        { "label": "k30", "matrix": [3, 0], "w": 1, "x": 0, "y": 3 },
        { "label": "k31", "matrix": [3, 1], "w": 1, "x": 1, "y": 3 },
        { "label": "k32", "matrix": [3, 2], "w": 1, "x": 2, "y": 3 },
        { "label": "k33", "matrix": [3, 3], "w": 1, "x": 3, "y": 3 },
        { "label": "k34", "matrix": [3, 4], "w": 1, "x": 4, "y": 3 },
        { "label": "k35", "matrix": [3, 5], "w": 1, "x": 5, "y": 3 },
        { "label": "k36", "matrix": [3, 6], "w": 1, "x": 6, "y": 3 },
        { "label": "k37", "matrix": [3, 7], "w": 1, "x": 7, "y": 3 },
        { "label": "k38", "matrix": [3, 8], "w": 1, "x": 8, "y": 3 },
        { "label": "k39", "matrix": [3, 9], "w": 1, "x": 9, "y": 3 },
        { "label": "k3a", "matrix": [3, 10], "w": 1, "x": 10, "y": 3 },
        { "label": "k3b", "matrix": [3, 11], "w": 1, "x": 11, "y": 3 }
```